build/transpile.js

```diff
@@ -310,6 +310,7 @@ class Transpiler {
             [ / = new /g, ' = ' ], // python does not have a 'new' keyword
             [ /console\.log\s/g, 'print' ],
             [ /process\.exit\s+/g, 'sys.exit' ],
+            [ /(while \(.*\)) {/, '$1\:' ], // While loops replace bracket with :
```

Comment:
That is not enough to transpile while-loops unfortunately, cause you also have to remove the closing bracket in Python.

Reply:
Yeah from I saw it looked like they were already removed, and all that was left was the opening bracket